src/api/core/index.ts

```diff
@@ -11,4 +11,20 @@ baseRequest.interceptors.request.use(
   (error) => Promise.reject(error),
 );
 
+const handleRequest = (config: AxiosRequestConfig) => config;
```

Comment:
proposal;

```suggestion
const handleRequest = <T>(config: AxiosRequestConfig<T>) => config;
```

<img src="https://user-images.githubusercontent.com/57402711/222076733-2c751607-7238-45c2-9ff9-04ffbcc3ee93.png" width="80%" />

`AxiosRequestConfig` 타입에 제네릭을 추가해줘야 하니 `handleRequest` 함수를 사용하는 부분에서 제네릭을 추가하면 어떨까요? (T로 네이밍한 제네릭은 타입 명칭을 변경해줘야겠네요….)